vault/resource_pki_secret_backend_intermediate_cert_request.go

```diff
@@ -183,6 +183,13 @@ func pkiSecretBackendIntermediateCertRequestResource() *schema.Resource {
 				ForceNew:      true,
 				ConflictsWith: []string{"managed_key_name"},
 			},
+			"add_basic_constraints": {
+				Type:        schema.TypeBool,
+				Description: "Whether to add a Basic Constraints extension with CA: true",
```

Comment:
nit: the description seems a bit of. I think something like `Set 'CA: true' in the 'Basic Constraints' extension.` is a bit more concise. Also it seem that this is only used as a work-around for AD Certificate Services. May want to include that as well.

Reply:
Agreed, and I think the description included on the docs page describes it a lot better.